src/js/vic-v2/helpers.jsx

```diff
@@ -156,6 +187,9 @@ export function submit(form, formConfig) {
     }).then(resp => {
       const guid = resp.data.attributes.guid;
       pollStatus(guid, response => {
+        window.dataLayer.push({
+          event: `${formConfig.trackingPrefix}-submission-successful`,
```

Comment:
Is this some sort of analytics thing?

Reply:
Yeah, it's how we send events to Google Analytics.